### ISSUE TITLE
option to select a part of lyrics

### ISSUE THREAD

**Issue by smjt2000** [TecCheck/FastLyrics#28]
Hi.
I'm not good in English.
I think it's a good ability/option to select any part of the lyrics and then I can copy it. Or other options that is possible like search or share the selected text.

**Comment by TecCheck** [comment]
Yes that is a good idea. Unfortunately there are some technical problems I have to solve before that works with synced lyrics. Thanks for the suggestion

**Comment by TecCheck** [comment]
You know it should already be possible to select the non synced lyrics and copy them :D

Could you try that out and tell me if it works on your device?

**Comment by smjt2000** [comment]
Yeah it's working.
When I post this issue, I wasn't on the last version, after update I can select and options are copy, share, select all, translate(google translate).
But when I open a lyrics and if I don't scroll down, it jumps down to where the thumbnail and song name not shown(goes down to do this) and selection is not a single word, it's from where I tried to select, to the position of my finger, but it's scrolled down.

As my English is not good and you maybe confused, here is the screen record:


https://github.com/TecCheck/FastLyrics/assets/60272086/faf1ddeb-9888-45d6-9716-0870a10abf6d

**Comment by smjt2000** [comment]
No, it's not a paragraph.
I tested again, it always select as the height of top bar(thumbnail and title).
